contracts/liquidators/LiquidationTriggerChainlinkMainAsset.sol

```diff
@@ -22,6 +22,8 @@ contract LiquidationTriggerChainlinkMainAsset is LiquidationTriggerBase, Reentra
     // uniswap-based oracle contract
     ChainlinkedOracleSimple public immutable chainlinkedOracleMainAsset;
```

Comment:
`ChainlinkedOracleMainAsset` has been fixed and now it does Q112-encode return values.
It's ok since this contract hasn't been deployed yet.
The contract in the external repo will be updated after approval of this PR.